src/components/MainPage.vue

```diff
@@ -116,4 +116,7 @@ export default {
 </script>
 
 <style>
+.v-expansion-panel-content__wrap {
+  padding: 5px !important;
+}
```

Comment:
I don't like this but surprisingly it seems that there is no other solution.
I think it should be declared in `Importers.vue` under `<style scoped>`